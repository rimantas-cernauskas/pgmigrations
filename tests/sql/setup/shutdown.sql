SELECT pgmigrations.revoke_privileges('__pgmigrations_testuser');
SELECT pgmigrations.revoke_privileges('__pgmigrations_superuser');
-- Remove test roles
DROP ROLE __pgmigrations_testuser;
DROP ROLE __pgmigrations_superuser;