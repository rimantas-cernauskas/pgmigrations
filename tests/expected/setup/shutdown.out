SELECT pgmigrations.revoke_privileges('__pgmigrations_testuser');
 revoke_privileges 
-------------------
 t
(1 row)

SELECT pgmigrations.revoke_privileges('__pgmigrations_superuser');
 revoke_privileges 
-------------------
 t
(1 row)

DROP EXTENSION pgmigrations;
-- Remove test roles
DROP ROLE __pgmigrations_testuser;
DROP ROLE __pgmigrations_superuser;
